roles/technical_principal.md

```diff
@@ -0,0 +1,154 @@
+# Technical Principal
+
+Our Technical Principals are consultative engineers who are commercially responsible for growing Made Tech both in terms of maintaining the quality of our teams as we scale and in helping win new business.
+
+## What does the job entail?
+
+Technical Principals sit across multiple delivery teams ensuring software engineering excellence and engaging in technical discovery work.
```

Comment:
In my head I'm seeing this role in less about ensuring software engineer excellence (for me that's more about code quality, which sits closer to lead/senior) and more about sanity of solution which is gluing together the business/technology piece